scripts/circleci-branch-update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+update-wrapper-node-version () {
+    version=$(<.nvmrc)
+    version=${version:1}
+    config=( $(jq -r '.circleCIToken' ./config/local-${NODE_ENV}.json) )
+    token=${config[0]}
+
+    curl --write-out %{http_code} --silent --output /dev/null -X POST --header "Content-Type: application/json" -d "{\"name\":\"NODE_VERSION\", \"value\":\"$version\"}" https://circleci.com/api/v1.1/project/github/automattic/wp-e2e-tests-for-branches/envvar?circle-token=${token}
```

Comment:
Maybe add a note to the README under wrapper repo's stating to add them here if you add a new one